app/models/user.py

```diff
@@ -0,0 +1,20 @@
+from sqlmodel import SQLModel, Field
+from sqlalchemy import UniqueConstraint
+
+from app.models.util import TimestampModel, UUIDModel
+
+
+class UserBase(SQLModel):
+    email: str = Field(max_length=255, nullable=False)
+    full_name: str = Field(max_length=255, nullable=False)
+
+
+class User(UUIDModel, TimestampModel, UserBase, table=True):
```

Comment:
What's the timestamp of?

Reply:
timestamp model is defined in `app.models.util.py`, and contains a factory that automatically adds the timestamp an entry was created at during the creation process based on local timezone. By inheriting from it, this functionality is passed to User. Timezone should potentially be standardized if plans are to create instances in multiple regions..